### DISCUSSION TITLE
How to get weather prediction data for a longer duration say for a month using the rest api

### DISCUSSION THREAD

**Discussion by pinky884** [weather-gov/api#268] [Q&A]
I am looking to get weather forecast for a period of 1 month using a rest api.
Currently I am using http://forecast.weather.gov/MapClick.php?lat=%s&lon=%s&FcstType=digitalDWML to get weather forecast data and this returns me the forecast data for 1 week from now.

Is it possible to get weather forecast data for 1 next month starting today? If yes, request you to kindly share the api details that can be used to get the same.

**Comment by JJKraw** [comment]
Are you looking for detailed forecast information beyond one week? If so, I don't think you are going to find it from the NWS because there is no way that data can be accurate enough to be useful. Even the forecast beyond three days often has a low level of confidence.

**Comment by StephenClouse** [comment]
^ This. We offer only data that is at least scientifically sound.

If you'd like the forecast a month or more out, consider a psychic or a tarot reading, or just roll some D&D dice. It won't be as scientific but it has about the same chance of being accurate :)
